test/user.test.js

```diff
@@ -205,6 +205,79 @@ describe('User', function() {
       });
     });
 
+    describe('Password length validation', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfItXYZ1';
+
+      it('rejects passwords longer than 72 characters', function(done) {
+        try {
+          User.create({ email: 'b@c.com', password: pass73 }, function(err) {
+            if (err) return done (err);
+            expect(err).to.have.property('message', /Password too long/);
+          });
+        } catch (e) {
+          done();
+        }
+      });
+
+      it('rejects a new user with password longer than 72 characters', function(done) {
+        try {
+          var u = new User({ username: 'foo', password: pass73 });
+          assert(false, 'Error should have been thrown');
+        } catch (e) {
+          done();
```

Comment:
Please verify that `e` is the error you are expecting, and not something like `TypeError: undefined is not a function`.